### ISSUE TITLE
sort in descending order

### ISSUE THREAD

**Issue by c64bob** [kollerlukas/Camera-Roll-Android-App#137]
I would love to have an option to sort in descending order. This has been mention in [issue 62](https://github.com/kollerlukas/Camera-Roll-Android-App/issues/62), but I don't think it has been implemented. Would it be possible to add that option?

**Comment by turbo2ltr** [comment]
This please.  Having newest at the top is default for most galleries.  Now I have to do a lot of scrolling to find the newest photos in large albums.

**Comment by btong** [comment]
I have a lot of photos and videos without exif data.

Currently I have to sort by filename, then scroll for ten seconds to the bottom. It makes using the app more painful than it could be, so this option would help me a lot. Thanks.

**Comment by turbo2ltr** [comment]
> scroll for ten seconds to the bottom
Yeah then you go one swipe too far and that ridiculous "feature" kicks in that sends you back to the main screen and you have to do it all over again.  Terrible UX.

**Comment by btong** [comment]
> Terrible UX.

That's not fair, it's a great piece of software. The result of a lot of work and we can use it for nothing. Thanks to the author.

**Comment by turbo2ltr** [comment]
This behavior is one of the main reasons I stopped using it.  It was incredibly frustrating to constantly be kicked back to the main menu for no reason other than I didn't stop scrolling past 1000 images at exactly the right moment.. So I would have to disagree with you about being "great". But that's just my opinion.  It definitely has potential to be great, but I won't use it until that feature can be turned off.
